docs/user/dashboard/aggregation-reference.asciidoc

```diff
@@ -1,10 +1,104 @@
 [[aggregation-reference]]
-== Aggregation reference
+== Feature comparison
```

Comment:
Can we call this something along the lines of `Supported features by panel type`?